packages/pangea-sdk/CHANGELOG.md

```diff
@@ -2,19 +2,27 @@
 
 All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+## [3.6.0] - 2024-01-11
+
+### Added
+
+- Structured encrypt support.
+
 ## [3.5.1] - 2023-12-22
 
-# Changed 
+### Changed
```

Comment:
Fixed all these h1's while I was here. It didn't make sense to have h1's after an h2 like this.